src/Integration/IntegrationInterface.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Sentry\Integration;
+
+/**
+ * This interface describes an Integration.
```

Comment:
I propose this description: `This interface defines a contract that must be implemented by middlewares, bindings or hooks that integrate certain frameworks or environments with the SDK.`